ReText/window.py

```diff
@@ -336,6 +343,9 @@ def __init__(self, parent=None):
 		self.editBar.addAction(self.actionCopy)
 		self.editBar.addAction(self.actionPaste)
 		self.editBar.addSeparator()
+		self.editBar.addAction(self.actionMoveUp)
+		self.editBar.addAction(self.actionMoveDown)
+		self.editBar.addSeparator()
```

Comment:
Is it really such a common action that it needs to be on the toolbar?

Reply:
Not really, I can remove it if you prefer